HW#5.py

```diff
@@ -0,0 +1,20 @@
+# Методы строк
+web_site = 'www.my_site.com#about'
+print(web_site.replace('#', '/'))
+print(web_site[:] + 'ing')
+print(" ".join('Ivanou Ivan'.split()[::-1]))
+print("   sad    ".strip())
+print("pARiS".capitalize())
+
+# Методы спиков
+print("Robin Singh".split())
+print("I love arrays they are my favorite".split())
+spisok = ['Ivan', 'Ivanou']
+stroka = 'Minsk Belarus'
+print(f"Привет, {spisok[0]} {spisok[1]}! Добро пожаловать в "
+      f"{stroka.split()[0]} {stroka.split()[1]}")
```

Comment:
better to split once:
```suggestion
stroka = stroka.split()
print(f"Привет, {spisok[0]} {spisok[1]}! Добро пожаловать в "
      f"{stroka[0]} {stroka[1]}")
```